CONTRIBUTING.md

```diff
@@ -302,12 +302,53 @@ about:
 
 1. [Push your changes] to your GitHub repository.
 
-### Finally...
+### Code review criteria & workflow
 
-Once enough people have reviewed and approved your work, it can be merged into
-the main branch of the main repository. Ask a member of the Checkbox team to do
-this. The branch should be then shortly automatically merged. The pull request
-status will then switch to “Merged”.
+As a general rule there is one reviewer per PR. Other reviewers may pop in to smooth up the process (where they feel confident they can _approve and merge_ changes after some were requested by another review). When more reviewers are needed an approval of, comments explaining that are made in the comment thread. Chiefly this is done to work around cases of low test coverage in cases when the changes affect something known to be of low quality (i.e. something significantly complex and hard to reason about, brittle, dated, known to have broken in the past, etc).
+
+The approving reviewer will merge the code if the review they choose to accept it. If they have chosen not to do that, they leave a comment explaining the rationale (mostly this exception is to cover situations when significant changes need to be staged across multiple releases).
```

Comment:
This is a bit hard to read to me, wdyt of: 

```suggestion
After accepting a change, the approving reviewer will merge the pull request. If they choose not to do that, they leave a comment explaining the rationale (mostly this exception is to cover situations when significant changes need to be staged across multiple releases).
```

Reply:
Good point, I tweaked your suggestion a bit and will commit in that shape.